### ISSUE TITLE
Email messages are not linking to images correctly

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/happy-heron#1717]

**Comment by amyehodge** [comment]
@jcoyne More details from what Irina Trapido is seeing are below. The important bit is the last image that shows that even after she unblocks images in the desktop app, they still don't display properly.

It looks like by default, all images are getting blocked:
Webmail:
![irina1.png](https://images.zenhubusercontent.com/5e9e06e81c23039800e23efc/338b8ba1-131e-47f6-aa69-07607d1e500f)

After I click "Show blocked content", they display fine:
![irina2.png](https://images.zenhubusercontent.com/5e9e06e81c23039800e23efc/d085c5ec-5d98-4406-b25a-dec037ae2ed2)

Outlook Desktop client also blocks images by default:
![irina3.png](https://images.zenhubusercontent.com/5e9e06e81c23039800e23efc/be5a1fe3-0c29-4b44-93f5-5ea941473e3d)

What is strange is that even after unblocking the images, the Stanford Libraries banner still doesn't show up in Outlook desktop app:
![irina3.png](https://images.zenhubusercontent.com/5e9e06e81c23039800e23efc/72283ac0-2a44-4754-8c42-da9989620ced)

**Comment by jcoyne** [comment]
@amyehodge I wonder if the outlook desktop client doesn't support SVG images.

**Comment by amyehodge** [comment]
@jcoyne I use the Outlook desktop client and the images have always come through fine for me.

**Comment by jcoyne** [comment]
@amyehodge Yes, it's possible it's only the Windows client that's the problem. Can we get someone else to test?

**Comment by amyehodge** [comment]
I'll see if I can find another Windows user. Going to be tough in our small pool of users.

**Comment by amyehodge** [comment]
One of the two people I'm talking to right now has confirmed Windows with Outlook desktop client.

**Comment by amyehodge** [comment]
The second person has confirmed Windows, and she uses both the Webmail with Chrome and the Outlook desktop client.

**Comment by jcoyne** [comment]
@amyehodge can you make a list of all the email clients and platforms you want to support. We can then make a plan for doing testing in each.

**Comment by amyehodge** [comment]
The list supported by UIT seems like a good one: https://uit.stanford.edu/emailcalendar/config#office365

**Comment by jcoyne** [comment]
Amy are you able to install each of those for testing?

**Comment by amyehodge** [comment]
I should be able to test the Android, iOS, and Mac clients. I would have to find someone else to test the Windows ones.

**Comment by justinlittman** [comment]
Per @amyehodge and @jcoyne, switch images to not-SVG.
